frontend/src/app/components/filters/query-filter/query-filter.component.html

```diff
@@ -16,6 +16,7 @@
     </label>
 
     <select required
+            *ngIf="valueType !== '[1]Boolean'"
```

Comment:
Please turn this check into a method on the `QueryFilterComponent` something like `operatorVisible(valueType)`, this will make it a bit more clear when the operator is hidden.

Reply:
yes that's a great idea. i will go through this.